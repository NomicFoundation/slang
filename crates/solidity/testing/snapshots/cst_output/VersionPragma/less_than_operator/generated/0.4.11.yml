# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Source: >
  1  │ solidity <1.0.0                                                                  │ 0..15

Errors: []

Tree:
  - VersionPragma (Rule): # 0..15 "solidity <1.0.0"
      - SolidityKeyword (Token): "solidity" # 0..8
      - VersionPragmaExpressionList (Rule): # 8..15 " <1.0.0"
          - VersionPragmaComparator (Rule): # 8..15 " <1.0.0"
              - LessThan (Token): "<" # 9..10
              - VersionPragmaSpecifier (Rule): # 10..15 "1.0.0"
                  - VersionPragmaValue (Token): "1" # 10..11
                  - Period (Token): "." # 11..12
                  - VersionPragmaValue (Token): "0" # 12..13
                  - Period (Token): "." # 13..14
                  - VersionPragmaValue (Token): "0" # 14..15
