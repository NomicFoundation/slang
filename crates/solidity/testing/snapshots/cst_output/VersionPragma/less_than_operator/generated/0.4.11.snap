# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Errors: 0

█ Rule(VersionPragma) - (0..15)                              │
├──► Token(Solidity) - (0..8)                                │ solidity
└──█ Rule(VersionPragmaSpecifiers) - (9..15)                 │
   └──█ Rule(VersionPragmaSpecifier) - (9..15)               │
      ├──► Token(VersionPragmaOperator) - (9..10)            │ <
      └──█ Rule(SeparatedVersionPragmaValues) - (10..15)     │
         ├──► Token(VersionPragmaValue) - (10..11)           │ 1
         ├──► Token(Period) - (11..12)                       │ .
         ├──► Token(VersionPragmaValue) - (12..13)           │ 0
         ├──► Token(Period) - (13..14)                       │ .
         └──► Token(VersionPragmaValue) - (14..15)           │ 0
